src/dappsExplorer/useOpenDapp.tsx

```diff
@@ -44,8 +46,14 @@ const useOpenDapp = () => {
     }
   }
 
-  const openDapp = (dapp: ActiveDapp) => {
-    ValoraAnalytics.track(DappExplorerEvents.dapp_open, getEventProperties(dapp))
+  const openDapp = (
+    dapp: ActiveDapp,
+    extraAnalyticsProperties: Record<string, string | number> = {}
```

Comment:
I saw your comment about having static types above and making it complex and brittle, but it feels we could at least have the 3 optional properties we want here.